config/routes.rb

```diff
@@ -1,11 +1,9 @@
 Rails.application.routes.draw do
+  resources :recipes
```

Comment:
You should limit just to index, show:
resources :recipes, only: [:index, :show]